File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Data Structures/Javascript/Excercises/program-5.js



// 5-Crear un conjunto con los números del 10 al 1 y mostrar el número más grande en el conjunto.

let conjunto = [8, 5, 2, 9, 6, 3, 7, 4, 1, 10]
let biggerN = 0;

for (let i = 0; i < conjunto.length; i++) {
    if (conjunto[i] > biggerN) {
        biggerN = conjunto[i];
    }
}
console.log(biggerN)

File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Data Structures/Python/Excercises/program-5.py
# 5-Crear un conjunto con los números del 10 al 1 y mostrar el número más grande en el conjunto.


conjunto = [10,9,8,7,6,5,4,3,2,1]

biggerN = 0;

for c in conjunto:
    if c > biggerN:
        biggerN = c;
print(biggerN)